All Core Devs Meetings/Meeting 117.md

```diff
@@ -15,13 +15,11 @@
 # **Summary:**
 
 - London fork is on Goerli 
-
-- client teams describe how they handle transaction pool sorting and price bump for transaction removal 
-
+- London Block on Mainnet is 12965000
+- Client teams describe how they handle transaction pool sorting and price bump for transaction removal 
 - [Merge Checklist is out](https://github.com/ethereum/pm/blob/master/Merge/mainnet-readiness.md)
-
+- Shanghai will happen sometime in november to push back the difficulty bomb but discussion is needed on what IF anything else will be added with it
```

Comment:
It was a part of the discussion and could be left out of the summary. The Merge vs Shanghai decision is to be continued.

Reply:
Based on our chat, I think you may ignore the above comment. Notes is good to go. Thanks @DockBoss!